content/blog/2018/export-logs-through-stackdriver.md

```diff
@@ -43,6 +43,7 @@ Common setup for all sinks:
 
 1. Enable [StackDriver Monitoring API](https://cloud.google.com/monitoring/api/enable-api) for the project.
```

Comment:
StackDriver -> Stackdriver

Please check the rest of the doc for the correct capitalization.